app/src/main/java/com/bunbeauty/fooddeliveryadmin/screen/menulist/cropimage/CropImageDefaults.kt

```diff
@@ -0,0 +1,30 @@
+package com.bunbeauty.fooddeliveryadmin.screen.menulist.cropimage
+
+import android.graphics.Color
+import com.bunbeauty.fooddeliveryadmin.R
+import com.canhub.cropper.CropImageOptions
+import com.canhub.cropper.CropImageView
+
+object CropImageDefaults {
+
+    fun options(): CropImageOptions {
+        return CropImageOptions(
+            imageSourceIncludeCamera = false,
+            cropShape = CropImageView.CropShape.RECTANGLE,
+            showProgressBar = false,
+            autoZoomEnabled = false,
+            aspectRatioX = 1000,
```

Comment:
lets extract magic numbers

Reply:
Is it magic numbers? 🤔
I don't think so